docs/firestore/usage.mdx

```diff
@@ -140,13 +140,24 @@ Once returned, you can subscribe to updates via the `listen()` method. The below
 which helps automatically manage the streams state and disposal of the stream when it's no longer used within your app:
 
 ```dart
-class UserInformation extends StatelessWidget {
+class UserInformation extends StatefulWidget {
   @override
-  Widget build(BuildContext context) {
-    CollectionReference users = FirebaseFirestore.instance.collection('users');
+    _UsetInfomation createState() => _UserInfomationState();
+}
+
+class _UserInformationState extends State<UserInformation> {
+  Stream<QuerySnapshot> _subscribe;
+  
+  @override
+  initState() {
+    super.initState();
+    _subscribe = FirebaseFirestore.instance.collection('users').snapshots();
+  }
```

Comment:
```suggestion
  final Stream<QuerySnapshot> _usersStream = FirebaseFirestore.instance.collection('users').snapshots();
```

Reply:
That does the same thing: creating the stream only once.

The `initState` is not necessary for this here

Reply:
Thank you! So then no need of StatefulWidget any more?